base_station/gui/src/components/DriveControls.vue

```diff
@@ -68,12 +69,24 @@ export default {
           const gamepad = gamepads[i]
           if (gamepad) {
             if (gamepad.id.includes('Logitech')) {
-              this.dampen = gamepad.axes[JOYSTICK_CONFIG['dampen']] * 0.5 + 0.5
+              // Make dampen 1 when slider is pushed forward, 0 when pulled back
+              // Raw value is -1 to 1
+              this.dampen = gamepad.axes[JOYSTICK_CONFIG['dampen']] * -0.5 + 0.5
+
+              // -1 multiplier to make turning left a positive value
+              let rotation = -1 * (gamepad.axes[JOYSTICK_CONFIG['left_right']] + gamepad.axes[JOYSTICK_CONFIG['twist']])
+              if (rotation > 1) {
+                rotation = 1
+              }
+              else if (rotation < -1) {
+                rotation = -1
+              }
 
               const joystickData = {
                 'type': 'Joystick',
-                'forward_back': gamepad.axes[JOYSTICK_CONFIG['forward_back']],
-                'left_right': gamepad.axes[JOYSTICK_CONFIG['left_right']],
+                // forward on joystick is -1, so invert
+                'forward_back': -1 * gamepad.axes[JOYSTICK_CONFIG['forward_back']],
```

Comment:
i think this double flips now bc of this PR https://github.com/umrover/mrover-workspace/pull/1013, i can revert it back there?

Reply:
Yeah you can, or I can push a commit changing that later today

Reply:
I think the GUI should be in charge of flipping the joystick value so that lcms sent over /drive_control are not reversed

Reply:
sg